addons/web/static/tests/fields/basic_fields_tests.js

```diff
@@ -1464,6 +1464,56 @@ QUnit.module('basic_fields', {
 
     });
 
+    QUnit.test('image fields in subviews are loaded correctly', function (assert) {
+        assert.expect(8);
+
+        this.data.partner.records[0].__last_update = '2017-02-08 10:00:00';
+        this.data.partner.records[0].document = 'myimage';
+        this.data.partner_type.fields.image = {name: 'image', type: 'binary'};
+        this.data.partner_type.records[0].image = 'product_image';
+        this.data.partner.records[0].timmy = [12];
+
+        var form = createView({
+            View: FormView,
+            model: 'partner',
+            data: this.data,
+            arch: '<form string="Partners">' +
+                    '<field name="document" widget="image" options="{\'size\': [90, 90]}"/>' +
+                    '<field name="timmy" widget="many2many">' +
+                        '<tree>' +
+                            '<field name="display_name"/>' +
+                        '</tree>' +
+                        '<form>' +
+                            '<field name="image" widget="image"/>' +
+                        '</form>' +
+                    '</field>' +
+                '</form>',
+            res_id: 1,
+            mockRPC: function (route, args) {
+                if (route === 'data:image/png;base64,myimage' || route === 'data:image/png;base64,product_image') {
+                    assert.ok(true, "should called the correct route");
+                    return $.when('wow');
+                }
+                return this._super.apply(this, arguments);
+            },
+        });
+
+        assert.strictEqual(form.$('tr.o_data_row').length, 1, 'There should be one record in the many2many');
```

Comment:
could you use the same style as other tests, i.e. limit to 80 (soft limit) or 100 (hard limit) characters? Typically, we write the last argument at the next line.